personio-personnel-data-api-oa3.yaml

```
@@ -79,6 +79,13 @@ paths:
           description: Find an employee with the given email address. The response is still a list, containing only the filtered employee.
           schema:
             type: string
+        - name: updated_since
+          in: query
+          required: false
+          description: Find an employees that had been updated since the provided value. The format is `ISO 8601` (2022-12-24T08:15:30).
```

Which is it, singular or plural?

```suggestion
          description: Find an employee that has been updated since the provided date. The format is `ISO 8601` (2022-12-24T08:15:30).
```

---

I think it's plural right? 

So it would be something like:
```suggestion
          description: Find employees that have been updated since the provided date. The format is `ISO 8601` (2022-12-24T08:15:30).
```